src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -93,19 +94,24 @@ const TopicPage = ({ pageData }) => {
               title: curationTitle,
               link,
               position,
+              visualStyle,
             }) => (
-              <Curation
-                headingLevel={curationTitle && 3}
-                key={curationId}
-                visualStyle={VISUAL_STYLE.NONE}
-                visualProminance={visualProminence}
-                promos={summaries}
-                title={curationTitle}
-                topStoriesTitle={topStoriesTitle}
-                position={position}
-                link={link}
-                curationLength={curations && curations.length}
-              />
+              <React.Fragment key={curationId}>
+                {visualStyle === 'BANNER' && (
```

Comment:
I just re-read the initial requirements, and we may need to include an additional condition for if visual prominence is Normal: 

>     For Message Banner, use visual settings as follows (in the BFF):
>         Visual style: Banner
>         Visual prominence: Normal 
> 
>     If visual setting does not match the above, then do not render content